core/src/main/java/org/akaza/openclinica/dao/login/UserAccountDAO.java

```diff
@@ -502,7 +502,7 @@ public Object getEntityFromHashMap(HashMap hm, boolean findOwner) {
         eb.setPasswdChallengeAnswer(passwdChallengeAnswer);
 
         // pull out the roles and privs here, tbh
-        if (!userName.contains(".")) {
```

Comment:
@jkeremian Why is setting roles skipped if username contains dot?

Reply:
@rooney @svadla-oc 
Hi Ron, Please try this and let me know 
Replace the following lines in UserAccountDAO.java with this code
line 490
        if (findOwner && !UserType.get(userTypeId.intValue()).equals(UserType.PARTICIPATE)) {


Line 505
        if (!UserType.get(userTypeId.intValue()).equals(UserType.PARTICIPATE)) {
This should take care of the issue where looking into the type of username for Participate without looking into actual username value content.